mrrc/pkgs/maven.py

```diff
@@ -238,6 +238,18 @@ def handle_maven_uploading(
         )
         logger.info("maven-metadata.xml uploading done")
 
+    # this step generates index.html for each dir and add them to file list
+    # index is similar to metadata, this will be updated everytime
+    index_files = valid_paths
+    if __META_FILE_GEN_KEY in meta_files:
+        index_files = index_files + meta_files[__META_FILE_GEN_KEY]
+    html_files = indexing.path_to_index(top_level, index_files, bucket)
+    logger.info("Start uploading index files to s3")
+    s3_client.upload_metadatas(
+        meta_file_paths=html_files, bucket_name=bucket, product=prod_key, root=top_level
```

Comment:
I don't think indexing meta files should contain product information in metadata, because it will overflow. For example, for the org/index.html, we can imagine there will be massive products contains this folder, so this index.html will contain all these product information in metadata if we set it. We don't know what's the limitation of the AWS metadata now, so we should avoid it.